Update file to orthogonalize ME-ICA regressors

kristinazvolanek opened the pull request:
Changes:
1. Incorporated code written by @n-reddy to automatically extract rejected and accepted components from the tedana output TSV files.
2. Added inputs to account for extra TRs on the CO2 regressor.

n-reddy reviewed (changes requested):
extraTR and nTR are currently hard-coded --> should be fixed

Two notes:
- In many cases, when using multi-band, we remove the first 10 TRs. This should be fine for interacting with this code, but could make a note that the CO2 output from the phys regressors code should have the first 10 TRs removed before inputting to this code; and the original number of TRs would then be the original number of TRs - 10
- The rejected.1D and accepted.1D files don't need to be made; could remove those lines of code

kristinazvolanek commented:
@n-reddy thank you!! I removed the hard-coded variables. In response to your other points:

- I added documentation explaining that users should remove volumes from their PETCO2hrf trace prior to running this script. Let me know if any of this is unclear.
- The 'rejected.1D' file is used in the 3DTproject command, and I don't think it hurts to output the non-transposed 'accepted.1D' components, so I left them in.

n-reddy reviewed (approved):
LGTM!
